booth.spec

```diff
@@ -40,7 +40,12 @@ BuildRoot:      %{_tmppath}/%{name}-%{version}-build
 BuildRequires:  asciidoc
 BuildRequires:  autoconf
 BuildRequires:  automake
+BuildRequires:  pkgconfig
+%if 0%{?suse_version}
 BuildRequires:  glib2-devel
+%else
+BuildRequires:  pkgconfig(glib-2.0)
+%fi
```

Comment:
On 18/04/16 08:21 -0700, Dejan Muhamedagic wrote:
>>  BuildRequires:  glib2-devel
>> +%else
>> +BuildRequires:  pkgconfig(glib-2.0)
>> +%fi
> 
> This should be %endif, right?

Right, thanks for noticing.
(I blame vim highlighter for showing "%if" the same way as "%fi").

-- 
Jan (Poki)